app/views/curation_concerns/base/_representative_media.html.erb

```diff
@@ -1,4 +1,4 @@
-<% if work.representative_id.present? && (fs = ::FileSet.load_instance_from_solr(work.representative_id)) %>
+<% if presenter.representative_id.present? && (fs = ::FileSet.load_instance_from_solr(presenter.representative_id)) %>
```

Comment:
I know this isn't germane to the PR, but it seems like the presenter ought to be responsible for returning a FileSet, instead of putting this into the view, but if that's your usual MO for this, it's fine.